tests/integration/json-schema-files/redis-schema-remote-entity.json

```diff
@@ -0,0 +1,1194 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "",
+  "properties": {
+    "events": {
+      "items": {
+        "properties": {},
+        "required": []
+      },
+      "type": "array"
+    },
+    "integration_version": {
+      "minLength": 1,
+      "pattern": "^1.2.0$",
+      "type": "string"
+    },
+    "data": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "entity": {
+            "minProperties": 2,
+            "properties": {
+              "name" : {
+                "type": "string"
+              },
+              "type": {
+                "type": "string"
+              }
+            }
```

Comment:
Probably it's better to add here: `"required": [ "name", "type"]` (In my initial schema they weren't.